### PR TITLE
[classifier] Added junction=circular.

### PR DESCRIPTION
**Pull request by vng**
Needed for https://github.com/organicmaps/organicmaps/issues/5468

### CODE REVIEW

**Review by biodranik — APPROVED** [review]

File: data/strings/types_strings.txt [hunk 1]
```diff
@@ -13366,7 +13366,7 @@
     uk = Перехрестя
     zh-Hans = 交叉口
 
-  [type.junction.roundabout]
+  [type.junction.circular]
     en = Roundabout
```
[the remark this answers is not in the document]
Comment (vng) [reply data/strings/types_strings.txt:13370]: Круговой перекрёсток ? :)

**Comment by matheusgomesms** [comment]
pt = Rotunda
pt-BR = Rotatória

**Comment by proxygeek** [comment]
Probably this has been discussed before but literal translations from English to local vernaculars such as Hindi (Hi) or Marathi (Mr) are technically correct but almost no-one uses these terms. This is especially true in spoken language. Not sure how to tackle that as ours is a text-UX and not a voice-UX. And yet I feel conflicted about the literal translations that are hardly understood or used.

Are there any agreed upon rule(s)-of-thumb here?

**Comment by biodranik** [comment]
> Are there any agreed upon rule(s)-of-thumb here?

Use the translation that most users will understand. Don't stick to theoretical "correctness", focus on the usability and clarity.

**Comment by proxygeek** [comment]
Perfect! Will do 👍🏽

**Pull request merged**